src/modules/sub-items/services/bulk.service.js

```diff
@@ -54,7 +55,7 @@ const processBulkResponse = (locations, successCode, callback, response) => {
 };
 
 const makeBulkRequest = ({ token }, requestBodyOperations, callback) => {
-    const request = new Request(Routing.generate('ezplatform.bulk_operation'), {
+    const request = new Request(ENDPOINT_BULK, {
         method: 'POST',
         headers: {
             ...HEADERS_BULK,
```

Comment:
I think in this case we should send `X-Siteaccess` in header.

Reply:
Yes, this is important.

Reply:
Good point!